x-pack/plugins/transform/public/app/lib/authorization/components/with_privileges.tsx

```diff
@@ -74,27 +74,31 @@ const MissingClusterPrivileges: FC<MissingClusterPrivilegesProps> = ({
   missingPrivileges,
   privilegesCount,
 }) => (
-  <EuiPageContent>
-    <NotAuthorizedSection
-      title={
-        <FormattedMessage
-          id="xpack.transform.app.deniedPrivilegeTitle"
-          defaultMessage="You're missing cluster privileges"
-        />
-      }
-      message={
-        <FormattedMessage
-          id="xpack.transform.app.deniedPrivilegeDescription"
-          defaultMessage="To use this section of Transforms, you must have {privilegesCount,
+  <EuiFlexGroup justifyContent="spaceAround">
+    <EuiFlexItem grow={false}>
+      <EuiPageContent verticalPosition="center" horizontalPosition="center" color="danger">
```

Comment:
❤️  Thanks for this PR @peteharverson ! 

QQ: I noticed that this error empty state is not vertically centered (in the screenshot) and **my guess** is that it has to do with the flex group/item wrapper. Does removing the them get it centered? Otherwise, I'd continue to look at any wrappers between the last `.euiPageContentBody` provided by the template, and your content.

Reply:
@cchaos I fixed the vertical centering of the error empty state in https://github.com/elastic/kibana/pull/102648/commits/3ec60374b218398e06b6a0ff1c35e14b2e6f3786 and updated the corresponding screenshot in the description.